rocAL/rocAL_pybind/examples/tf_petsTrainingExample/train_withROCAL_withTFRecordReader.py

```diff
@@ -204,17 +201,17 @@ def main():
 							if predicate == True:
 								num_correct_predicate += 1
 						print ("Step :: %s\tTarget :: %s\tPrediction :: %s\tCorrect Predictions :: %s/%s\tValidation Loss :: %.2f\tValidation Accuracy :: %.2f%%\t" % (j, val_target, val_prediction, num_correct_predicate, len(correct_predicate), val_loss, (val_accuracy * 100)))
+					valIterator.reset()
```

Comment:
This is a training example so we are calling reset for both train and val loaders once all the data is consumed. Since validation happens 100 steps, the val loader is fully iterated and then reset() is called to be used for the next val epoch